crates/accelerate/src/synthesis/linear/pmh.rs

```diff
@@ -159,7 +159,16 @@ pub fn synth_cnot_count_full_pmh(
     section_size: Option<i64>,
 ) -> PyResult<CircuitData> {
     let arrayview = matrix.as_array();
-    let mut mat: Array2<bool> = arrayview.to_owned();
+    let mat: Array2<bool> = arrayview.to_owned();
+    let num_qubits = mat.nrows();
+
+    let instructions = _synth_cnot_count_full_pmh(mat, section_size);
+    CircuitData::from_standard_gates(py, num_qubits as u32, instructions, Param::Float(0.0))
+}
+
+type Instructions = (StandardGate, SmallVec<[Param; 3]>, SmallVec<[Qubit; 2]>);
+pub fn _synth_cnot_count_full_pmh(mat: Array2<bool>, sec_size: Option<i64>) -> Vec<Instructions> {
```

Comment:
1> Done!
2> I decided to keep it as `Option<i64>` because we have to convert from `Options<i64>` to `Option<usize>` in both occurrences in `pmh.rs` and `cnot_phase_synth.rs` as soon as we get data from python, and then send it to `synth_pmh`, furthermore its being converted to `usize` anyway in line: 185. Also no reason behind renaming it, this time I forgot to revert the name back to `section_size`, will do it in next commit! :) 

3> Done!